cognitedata/reveal
# chore: use yarn 4 for examples

#### Type of change

## Description :pencil:

Sets the package manager for examples to be the same as for the viewer.


## Test instructions :information_source:

1. Go to examples
2. Run yarn && yarn run start
3. It should work
